docs/workflow/blocks/http.rst

```diff
@@ -50,7 +51,8 @@ For simple requests, the ``endpoint`` can just be a simple string:
     }
 
 
-If the endpoint needs to be constructed from data, the endpoint can be specified as an object with a "valueGetter" attribute. 
+**Dynamic data** If the endpoint needs to be constructed from data, the endpoint can be specified as an object with a "valueGetter" attribute.
+"valueGetter" get data just from context.  
```

Comment:
Better English would be - 

"valueGetter" can only get data from the context.

Reply:
Thanks!